server/sonar-server/src/main/java/org/sonar/server/settings/ws/ValuesAction.java

```diff
@@ -0,0 +1,291 @@
+/*
```

Comment:
the format of the response should be cleaner. For a simple value the fields `values` and `setValues` should not appear in the response. To do that, you must encapsulate the repeated values in a specific Values proto message and you should not set it if there's no value